lib/pure/uri.nim

```diff
@@ -54,6 +54,7 @@ type
 
   UriParseError* = object of ValueError
 
+const unsafeUrlBytesToRemove = {'\t', '\r', '\n'}
```

Comment:
Suggestion: can you add a link to the WHATWG spec that describes these characters as a comment here? :)